sed/core/logging.py

```diff
@@ -19,26 +21,55 @@
 
 def setup_logging(
     name: str,
-    user_log_path: str = DEFAULT_LOG_DIR,
+    set_base_handler: bool = False,
+    user_log_path: str | None = None,
 ) -> logging.Logger:
     """
     Configures and returns a logger with specified log levels for console and file handlers.
 
     Args:
         name (str): The name of the logger.
-        user_log_path (str): Path to the user-specific log directory. Defaults to DEFAULT_LOG_DIR.
+        base_name (str, optional): The name of the base logger. Defaults to "sed".
```

Comment:
This arg is not present. I believe you mean set_base_handler

Reply:
True, needs to be updated

Reply:
done